packages/react-server/core/ReactServerAgent/Request.js

```diff
@@ -22,7 +22,8 @@ function Request(method, urlPath, cache) {
 	this._headers = {};
 	this._timeout = null;
 	this._type = "json"; // superagent's default, only for POST/PUT/PATCH methods
-
+	// CORS off by default
+	this._withCredentials = false;
```

Comment:
I think it's best to stick with  the same syntax that [superagent uses](https://github.com/visionmedia/superagent/blob/f5e39b7746268901c591dde08cd68a5e3944ced1/lib/request-base.js#L396)